### ISSUE TITLE
Database

### ISSUE THREAD

**Issue by FlorianStrobl** [Discord-Coding-Community/Pylon-Bot#3]
Hi! Nice that you use my Database code, but I think you copied the wrong code XD

You copied this code: https://github.com/FlorianStrobl/Discord-Pylon-Bot/blob/master/Scripts/Functions/WarnCommandWDatabase.ts
But this is an **example**. You want probably: **https://github.com/FlorianStrobl/Discord-Pylon-Bot/blob/master/Scripts/Functions/Database.ts**
You did this mistake to **every db folder** xD

**Comment by PhantomNimbi** [comment]
Yeah that was definitely my mistake. Thanks for the notice. Also soz for the late reply. I only just saw this message. I've pretty much already started using your betterKV version since this one lol.
